ibm_catalog.json

```diff
@@ -101,6 +97,162 @@
                     "value": "us-east"
                   }
                 ]
+              },
+              {
+                "key": "existing_cos_kms_key_crn",
+                "required": true
```

Comment:
Since we are marking this as required for catalog, can you please update the variable description in the variables.tf file (like we did in this [PR](https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/pull/62)) to make it clear that its actually optional. Suggest this:

> (OPTIONAL) The CRN of an existing KMS key to be used to encrypt the Cloud Object Storage buckets created by this solution. If no value is passed, a value must be passed for either the `existing_kms_instance_crn` input variable if you want to create a new key ring and key, or the `existing_log_archive_cos_bucket_name` and `existing_at_cos_target_bucket_name` input variables if you want to use existing buckets.